apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -241,6 +241,11 @@
                             "description" : "Number of acceptors in internal pool",
                             "type" : "integer",
                             "default" : 10
+                        },
+                        "debug_endpoints" : {
+                            "description" : "Enable (true) debug api. Disabled (false) by default",
```

Comment:
What constitutes part of the "debug api"? The paths tagged as `debug` in swagger.yaml, or the paths starting with `/debug`? It may not be immediate for the reader, and may lead to inconsistencies in the future.

Reply:
I just faced that issue, there are the two options mentioned by @lucafavatella above, and now this one (new) ?

Reply:
Description updated. I'm checking `debug` tag in swagger: https://github.com/aeternity/epoch/pull/1901/files#diff-744f44aa1c55e60ba2c122893c02ebe0R37